docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+    container_name: cinema_app
+    command: >
+      sh -c "python manage.py wait_for_db &&
+             python manage.py migrate &&
+             python manage.py collectstatic --noinput &&
+             python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - .:/app
```

Comment:
Mounting the entire project directory as a volume can lead to issues, especially in production. Ensure your `.dockerignore` is properly configured to exclude unnecessary files and directories.